app/internal/page/dependency_health_check.go

```diff
@@ -0,0 +1,26 @@
+package page
+
+import (
+	"net/http"
+
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/date"
+	"github.com/ministryofjustice/opg-modernising-lpa/internal/uid"
+)
+
+func DependencyHealthCheck(logger Logger, uidClient UidClient) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		_, err := uidClient.CreateCase(r.Context(), &uid.CreateCaseRequestBody{
+			Type: LpaTypePropertyFinance,
+			Donor: uid.DonorDetails{
+				Name:     "Jane Smith",
+				Dob:      uid.ISODate{Time: date.New("2000", "1", "2").Time()},
+				Postcode: "B147ED",
+			},
+		})
+
+		if err != nil {
+			logger.Print(err)
+			w.WriteHeader(http.StatusInternalServerError)
+		}
+	}
+}
```

Comment:
I've not added any tests to call this just yet as I'm thinking ahead to when prod is plumbed in we don't want to be creating dummy data in Sirius. I'm going to suggest a ping endpoint in the UID service that will at least give us confidence our signing and env config is correct without having to create data. Once that's working we can swap this out for a call to the new endpoint.